demos/connector/neo4j/cluster-gcn-on-cora-neo4j-example.ipynb

```diff
@@ -153,6 +153,24 @@
     "test_targets = target_encoding.transform(test_labels)"
```

Comment:
<p>These tiny clusters are a bit peculiar, especially because <code>q=1</code>,  so they're being handled one at a time. This means our GCN model is spending a big chunk of time being trained on subgraphs of size 1 or 2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
yeah its unfortunate, also the way the `q` parameter is setup it has to exactly divide the number of clusters which is 173.... which is prime....